### ISSUE TITLE
Sorting crashes

### ISSUE THREAD

**Issue by rfossella** [wenzhixin/bootstrap-table#624]
Hi
I'm using v1.6.0.
I'm sorting a column of that includes "time" that is created using moment.js.  One of the 'time' columns sorts fine; the other fails on **aa.localeCompare(bb)**.  Seems that the data is correct.  I do included a formatter that checks for null or empty string
````
function tbTimeFormatter(aValue) 
 {
    var newValue = aValue;
    if (aValue == null || aValue.trim() == '')
        newValue = 0;
    return  moment({seconds:newValue}).format("hh:mm:ss a");
}
````
aa=number
bb=""
Any idea?  Thank you!

![image](https://cloud.githubusercontent.com/assets/1350182/6680123/a457f9ce-cc29-11e4-95b7-4e3cbf80572c.png)

**Comment by wenzhixin** [comment]
Can you provide a jsFiddle with some data?

**Comment by wenzhixin** [comment]
Please update to the latest version, I think this bug has fixed.

**Comment by rfossella** [comment]
Thank you!  I will update ASAP and confirm.

**Comment by rfossella** [comment]
I have confirmed that this issue resolved in 1.7.0.  Thank you!
